package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "node build/dev-server.js",
     "build": "node build/build.js",
     "prod": "npm run build && npm start",
-    "start": "./node_modules/.bin/http-server -p ${BACKOFFICE_PORT:=3000} dist/",
+    "start": "./node_modules/.bin/start --port ${BACKOFFICE_PORT:=3000}",
```

Comment:
Well... I checked, it's actually `KUZBO_port`, which is ugly (case mess-up). I'd like to set the app name to `kuzbo` instead. Is it ok to have env vars in lowercase?